recipes/opentracing-cpp/all/patches/0001-Export-mocktracer-dll.patch

```diff
@@ -0,0 +1,18 @@
+# They forgot to export the mocktracer dll
+---
+ mocktracer/CMakeLists.txt | 1 +
+ 1 file changed, 1 insertion(+)
+
+diff --git a/mocktracer/CMakeLists.txt b/mocktracer/CMakeLists.txt
+index 9835fe3..c245fef 100644
+--- a/mocktracer/CMakeLists.txt
++++ b/mocktracer/CMakeLists.txt
+@@ -21,6 +21,7 @@ if (BUILD_SHARED_LIBS)
+   install(TARGETS opentracing_mocktracer
+           COMPONENT DIST
+           EXPORT OpenTracingTargets
++          RUNTIME DESTINATION ${LIB_INSTALL_DIR}
```

Comment:
Usually executables are installed in `bin/`

```suggestion
+          RUNTIME DESTINATION ${BIN_INSTALL_DIR}
```

Reply:
I know. I set it like this to be consistent with the main library that installs the `.dll` al `lib/`.

Reply:
It was added like this in master branch, but `${LIB_INSTALL_DIR}` for runtime is not standard. A patch should be submitted to upstream.

Reply:
Sorry, I missed this comment. I just uploaded changes for this